.circleci/config.yml

```diff
@@ -222,7 +222,7 @@ jobs:
           name: Tests TypeScript definitions
           command: pnpm typescript:ci
           environment:
-            NODE_OPTIONS: --max-old-space-size=1536
+            NODE_OPTIONS: --max-old-space-size=2048
```

Comment:
IMHO, we can revert it to where it was before pnpm migration PR.
Given that https://github.com/mui/mui-x/pull/12920 seems to solve the OOM issue.

Reply:
```suggestion
            NODE_OPTIONS: --max-old-space-size=1536
```